spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -131,7 +167,7 @@ module.exports = function (provider, setup) {
 
         // Wait a second before unpausing.  Data received in this second
         // will fail the expectation that paused is false in gotMessageAsync.
-        setTimeout(function() {
+        setTimeout(function () {
```

Comment:
Note that pause() on most platforms returns before actually guaranteeing that the socket is paused ... so you'd still want a timeout after calling pause :-(.